docs/en/latest/deployments/ack.md

```diff
@@ -44,14 +44,15 @@ The script below installs APISIX and the ingress controller:
 helm repo add apisix https://charts.apiseven.com
 helm repo add bitnami https://charts.bitnami.com/bitnami
 helm repo update
-kubectl create ns ingress-apisix
 helm install apisix apisix/apisix \
   --set gateway.type=LoadBalancer \
   --set ingress-controller.enabled=true \
   --set etcd.persistence.storageClass="alicloud-disk-ssd" \
   --set etcd.persistence.size="20Gi" \
+  --create-namespace \
   --namespace ingress-apisix \
-  --set ingress-controller.config.apisix.serviceNamespace=ingress-apisix
+  --set ingress-controller.config.apisix.serviceNamespace=ingress-apisix \
+  --set ingress-controller.config.apisix.adminAPIVersion=v3
 kubectl get service --namespace ingress-apisix
```

Comment:
We may add a note here to say we use APISIX 3.0 as the example so we set the `adminAPIVersion` to `v3`. If you're using APISIX 2.x, please reset it to `v2`.

Reply:
@tokers Should it look like this?

```bash
#  We use Apisix 3.0 as the example. If you're using Apisix v2.x, please set to v2
ADMIN_API_VERSION=v3
helm install apisix apisix/apisix \
  --set gateway.type=LoadBalancer \
  --set ingress-controller.enabled=true \
  --set etcd.persistence.storageClass="alicloud-disk-ssd" \
  --set etcd.persistence.size="20Gi" \
  --create-namespace \
  --namespace ingress-apisix \
  --set ingress-controller.config.apisix.serviceNamespace=ingress-apisix \
  --set ingress-controller.config.apisix.adminAPIVersion=$ADMIN_API_VERSION
```

Reply:
@anhldbk good idea! I think that's fine.  
cc @navendu-pottekkat

Reply:
@tao12345666333 @tokers @navendu-pottekkat Updated. PTAL